src/main/java/com/rental/rentalApp/RentalAppApplication.java

```diff
@@ -2,6 +2,12 @@
 
 import com.rental.rentalApp.entities.Client;
 import com.rental.rentalApp.repositories.ClientRepository;
+import com.rental.rentalApp.entities.Color;
+import com.rental.rentalApp.repositories.ColorRepository;
+import com.rental.rentalApp.entities.Brand;
+import com.rental.rentalApp.repositories.BrandRepository;
+import com.rental.rentalApp.entities.Model;
+import com.rental.rentalApp.repositories.ModelRepository;
```

Comment:
I believe this should work

```suggestion
import com.rental.rentalApp.entities.*;
import com.rental.rentalApp.repositories.*;
```

Reply:
Thats a good idea.